nilearn/plotting/img_plotting.py

```diff
@@ -644,6 +651,11 @@ def plot_roi(roi_img, bg_img=MNI152TEMPLATE, cut_coords=None,
             Lower bound for plotting, passed to matplotlib.pyplot.imshow
         vmax : float
             Upper bound for plotting, passed to matplotlib.pyplot.imshow
+        resampling_interpolation : str
+            Interpolation to use when resampling the image to the destination
+            space. Can be "continuous" to use 3rd-order spline interpolation,
+            or "nearest" (default) to use nearest-neighbor mapping.
```

Comment:
note that the default for `plot_roi` is different from the other 3 img_plotting functions, not sure if that's intentional but I haven't changed that behavior